### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Task_3/

### CODE REVIEW

File: html/Task_3/style.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  box-sizing: border-box;
+}
+
+.container {
+  width: 40%;
+  min-width: 450px;
+  margin: 0 auto;
+  padding: 20px;
+  background: lightgray;
+  font-family: "Lato", sans-serif;
```
Comment (tsvetkovv) [inline html/Task_3/style.css:13]: Good practice to set font family for root html element (or body)

File: html/Task_3/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  opacity: 0.7;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+}
+
+.row-1 span {
```
Comment (tsvetkovv) [inline html/Task_3/style.css:24]: Just `.row-1` 
Try to avoid increase specificity if it is not needed

File: html/Task_3/style.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  font-size: 8em;
```
Comment (tsvetkovv) [inline html/Task_3/style.css:25]: It's better to use `rem`and set base root size to have opportunity to scale all text by change base font

**Review by tsvetkovv — APPROVED** [review]
See comments 👀

**Review by tkachukivan — APPROVED** [review]

**Pull request merged**